### ISSUE TITLE
Queue is created with SSE enabled on first run

### ISSUE THREAD

**Issue by fmasa** [terraform-aws-modules/terraform-aws-sqs#44]
## Description

On the first apply, queue (in our case FIFO queue) is created with Server Side Encryption enabled (using SQS managed key) even though the default should be disabled SSE.
Re-applying the plan detects drift and disables SSE. 

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

- Module version [Required]: 3.4.2

- Terraform version: 1.3.4
- Provider version(s): 1.3.4

## Reproduction Code [Required]

```terraform
module "my_queue" {
  source  = "terraform-aws-modules/sqs/aws"
  version = "3.4.2"

  name = "my-queue.fifo"
  
  fifo_queue = true
  content_based_deduplication = true
}
```

## Expected behavior

SQS queue with disabled SSE is created.

## Actual behavior

SQS queue with SSE enabled (using SQS managed key) is created.

**Comment by fmasa** [comment]
No need to close this issue IMO, as it's being handled in #45

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.0.0 :tada:
